lib/plug/upload.ex

```diff
@@ -161,7 +161,7 @@ defmodule Plug.Upload do
   defp i(integer), do: Integer.to_string(integer)
 
   defp path(prefix, tmp) do
-    {_mega, sec, mili} = :erlang.now
+    {_mega, sec, mili} = :os.timestamp
```

Comment:
@MSch that would be fine, yeah. We also have retries in case two processes in the same scheduler do something in the same microsecond (high unlikely).